tests/hibernate6/hibernate6-reactive-mysql/src/test/java/example/hibernate6/reactive/MySQLApp.java

```diff
@@ -22,5 +22,6 @@
 @MicronautTest(transactional = false)
 @Property(name = "jpa.default.properties.hibernate.connection.db-type", value = "mysql")
 @Property(name = "jpa.default.reactive", value = "true")
+@Property(name = "test-resources.containers.mysql.image-name", value = "mysql:8.4.5")
```

Comment:
Unrelated to this PR, mysql:latest docker image was failing to start in tests so needed to update this 8.x